app/src/main/res/values/styles.xml

```diff
@@ -6,6 +6,9 @@
         <item name="colorPrimary">@color/colorPrimary</item>
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorAccent</item>
+
+        <item name="windowActionBar">false</item>
+        <item name="windowNoTitle">true</item>
```

Comment:
These aren't used and can be deleted I think

Reply:
Nah it's used behind the scenes to make the toolbar run